code/src/components/Labyrinth.js

```diff
@@ -0,0 +1,162 @@
+import React from "react"
+import { useSelector, useDispatch } from "react-redux"
+import { MiniMap } from "./MiniMap"
+import { nextStep } from "reducers/game"
+import styled from "styled-components/macro"
+
+
+import arrow from "../assets/redArrow.svg"
+
+const Section = styled.section`
+position:relative;
+width: fit-content;
+height: 100%;
+max-width: 300px;
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+margin: 100px auto 0 auto;
+
+`
+const Title = styled.h1`
+
+text-align: center;
+font-size: 0.8rem;
+background-color: rgba(0,0,0, 0.5);
+padding: 10px;
+margin:auto;
+border-radius:10px;
+
+
+@media (min-width: 768px) {
+    font-size: 1rem;
+  }
+@media (min-width: 1000px) {
+    font-size: 1.2rem;
+  }
+
+`
+const NorthButton = styled.button`
+position: absolute;
+display: flex;
+flex-direction: column;
+width: 30px;
+background: transparent;
+border: none;
+color: white;
+top: -30px;
+left: 50%;
+transform: rotate(0);
+`
```

Comment:
Since you're using the same styling and same image for the direction buttons, you could change the positioning and the turning of the buttons using props and then just have only one button component. Using the props could look something like this: 

`transform: ${props => {
props.direction === "North"
? "rotate(0)"
: props.direction === "South"
? "rotate(180)" 
`
And after using the props in the styled component here you would then pass the props into the button inside the return, see comment there ->